core/src/test/java/kafka/test/ClusterTestExtensionsTest.java

```diff
@@ -251,8 +251,7 @@ public void testClusterAliveBrokers(ClusterInstance clusterInstance) throws Exce
         Assertions.assertTrue(clusterInstance.brokers().containsKey(0));
     }
 
-
-    @ClusterTest(types = {Type.ZK, Type.CO_KRAFT, Type.KRAFT}, brokers = 4)
+    @ClusterTest(types = {Type.ZK, Type.CO_KRAFT, Type.KRAFT}, brokers = 4, listener = "TEST_LISTENER")
```

Comment:
why we need this change?

Reply:
In the beginning I just want to show an example about how to set listener in annotation. But that's not needed, I will remote in the next commit. Thanks.